examples/detailedlist/detailedlist/app.py

```diff
@@ -26,7 +35,6 @@ async def on_refresh_handler(self, widget, **kwargs):
         self.label.text = "List was refreshed."
 
     def on_delete_handler(self, widget, row, **kwargs):
-        self.label.text = f"Row {row.subtitle} is going to be deleted."
```

Comment:
Because deleting a row also changes the selection, this message was never visible.